thoughts-of-a-rustacean-learning-go.md

```diff
@@ -0,0 +1,244 @@
+<!--
+headline: Thoughts of a Rustacean learning Go
```

Comment:
To give even more context, how do you feel about the title:

### A Rust Contributor Tries Their Hand at Go

Or something along those lines?